wordpress-mobile/WordPress-iOS
# Update Jetpack logo

**Fixes** #8279 

**To test:**

Check that the Jetpack logo is the new one on the following screens:

1. Last paged image in Log In (animation)
2. Stats sign up prompt
3. Activity Log
4. Empty notifications state with no .com account (you'll have to comment code to make this happened, I've already tested it, you can trust me)

Before screenshots can be found for reference for 1 and 2 on the original issue.
